### PR TITLE
ContactComponent imprime datos en consola

### PR DESCRIPTION
**Pull request by PalomaRmMz**
### Changes made
Add class ES6 in ContactComponent
Check the state in inputs
Print content of inputs in console from ContactComponent

### Where to start
src\contact\contact-component.js

### How to check the changes?
Write text in the contact form and push "Enviar" button and open "Developer tools"

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -12,6 +13,7 @@ function App() {
       <div className='container'>
         <Route exact path="/" component={HomeComponent} />
         <Route path="/contact" component={ContactComponent} />
+        <Route path="/loader" component={LoaderComponent} />
```
Comment (NemeSnz) [inline src/App.js]: Theres no need to route this component

File: src/example-gql/example-gql-component.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
+import { LoaderComponent } from '../loader/loader-component';
```
Comment (NemeSnz) [inline src/example-gql/example-gql-component.js]: this file does not need to be tracked

File: src/loader/loader-component.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
+import './loader-style.css';
```
Comment (NemeSnz) [inline src/loader/loader-component.js]: Are you using this component at all?

File: src/screen-error/screen-error-component.js [hunk 4]
```diff
@@ -0,0 +1,16 @@
+import './screen-error-style.css';
```
Comment (NemeSnz) [inline src/screen-error/screen-error-component.js]: are you using this component?

**Review by NemeSnz — CHANGES_REQUESTED** [review]
It seems like you committed files that rae not being used at all

**Comment by PalomaRmMz** [comment]
> It seems like you committed files that rae not being used at all

ExampleGQLComponent, LoaderComponent and ScreenErrorComponent has been delete, will not be used anymore

**Review by NemeSnz — APPROVED** [review]

**Pull request merged**